include/taskolib/Sequence.h

```diff
@@ -529,6 +540,20 @@ class Sequence
      */
     void set_label(gul14::string_view label);
 
+    /**
+     * Add one or more maintainers to the sequence. You are free to choose what ever you
+     * can use to identify the maintainer. You can also type more than one maintainer
+     * where they can be separated with comma or semicolon. For example:
+     *
+     *      "John Doe john.doe@universe.org; Bob Smith boby@milkyway.edu"
+     *
+     * \param maintainers  One ore more maintainers of the sequence.
+     *
+     * \exception Does not accept control characters in the maintainer names itself.
```

Comment:
I guess we should mention which kind of exception is being thrown: `\exception Error is thrown if ...`
The comment about the trimming does not really have anything to do with exceptions, does it?